README.md

```diff
@@ -1,64 +1,62 @@
+ip address show wlan0 | grep inet | awk '{ print $2; }' | sed 's/\/.*$//'
 ```
-System: PostgreSQL
-Server: db
-Username: postgres
-Password: postgres
-Database: memoires-info
-```
-### Postgrest
-Postg rest is an api used to push and pull data from the database
-get request can be made to pull data from the database and post request can be made to push data to the database
-You can look at the database's content by going to `localhost:3000/data` in your web browser
 
-## Esp32
-### Installation
-This project uses esp home to manage the esp32 wich is run in a docker container
-Inside the config folder, start by copying the `secret.example.yaml` file to `secret.yaml` and fill in credentials
-Then to start the esp32, you can run the following command
+[Info]: <> (
+	This is the list of all the files that require to know the ip address of the computer
+)
+You then want to copy the ip address and paste it the following files with the port 3000:
+- esp32/config/secrets.yaml
+- Interface/src/contexts/SampleContext.tsx
+- login/.env
+
+---
+Now you want to generate a secret key for the jwt tokens used by the api
 ```bash
-cd esp32
-docker-compose up
+tr -dc A-Za-z0-9 </dev/urandom | head -c 32; echo
 ```
-### Home assistant
-You can access home assistant by going to `localhost:6052` in your web browser
-### Add esp32 to home assistant
-Tou can then add the esp32 to home assistant by pluging the esp32 to your computer that runs the docker
-Then press the `+ ADD DEVICE` button inside home assistant
-Then press continue and name your device, give him a clever name so you know wich device it is
-Then press `next` and select `ESP32` (on the top of the list)
-Once that's done you want to skip the installation you'll have to reinstall it later anyway
-Go to the card of the device and press `EDIT`
-Here you want to copy everything from the line 31 of the `esp32/esp32.yaml` and paste it at the end of your esp32's configuration file
-Then press `INSTALL` and select `Plug into the computer running ESPHome Dashboard` (the 3rd option)
-You then want to look for the device for wich the path is `/dev/ttyUSB0` and select it
-Now the installation should start and you should see the logs of the installation
-It can take a few minutes to compile and install the firmware, be patient
-Once the installation is done, you can press `STOP` to exit the logs
-You can now unplug the esp32 the computer and plug it to any other power source
-If you need to change the configuration of the esp32, you can now do it wirelessly from the home assistant interface
+
+[Info]: <> (
+	This is the list of all the files that require to know the jwt secret key
+)
+You then want to copy the secret key and paste it the following files:
+- login/.env
+- docker-compose.yml
+
+---
+Finally you can fill the `esp32/config/secrets.yaml` with the credentials of the wifi\
+Once that's done you can create a postgres database called `memoires-info` from the `psql.sql` file
```

Comment:
Db creation should be done with the POSTGRES_DB en Var. See doc https://hub.docker.com/_/postgres
See also the "Initialization scripts" section to pick psql.sql on the first execution.